components/script/dom/formdata.rs

```diff
@@ -93,6 +93,18 @@ impl FormDataMethods for FormData {
                  })
     }
 
+    // https://xhr.spec.whatwg.org/#dom-formdata-getall
+    fn GetAll(&self, name: DOMString) -> Vec<FileOrString> {
+        self.data.borrow()
+                 .get(&name)
+                 .map_or(vec![], |data|
```

Comment:
I probably would have done fancy tricks with `flat_map` here but this works too :smile: